sdks/python/apache_beam/typehints/opcodes.py

```diff
@@ -261,13 +264,22 @@ def build_list(state, arg):
 
 
 def load_attr(state, arg):
+  """Replaces the top of the stack, TOS, with with
+  getattr(TOS, co_names[arg])
+  """
   o = state.stack.pop()
   name = state.get_name(arg)
   if isinstance(o, Const) and hasattr(o.value, name):
     state.stack.append(Const(getattr(o.value, name)))
   elif (isinstance(o, type)
```

Comment:
So I have a small improvement here that's in 39b21bfa22bf8fbf5107bdff354252be3ca75cd2 if you want to include it (but optional)

Reply:
I was not able to handle the nested class case so I stayed with a subset of callable types.